client/.eslintrc.js

```diff
@@ -23,5 +23,5 @@ module.exports = {
         sourceType: 'module',
     },
     plugins: ['react'],
-    rules: {},
+    rules: { 'react/prop-types': 0 },
```

Comment:
I ended up adding the prop-types package and implementing it on some components in my branch. I'm good either way though.